src/theory/bv/bitblast/eager_bitblaster.cpp

```diff
@@ -262,6 +265,35 @@ bool EagerBitblaster::collectModelInfo(TheoryModel* m, bool fullModel)
       }
     }
   }
+
+  // Collect the values for the Boolean variables
+  std::vector<TNode> vars;
+  d_cnfStream->getBooleanVariables(vars);
```

Comment:
Careful, this returns ALL Boolean variables, right?

Reply:
No, we've discussed this offline and we have discovered that it only returns Boolean variables that appear in the input and none of the Boolean variables that are generated during bit-blasting. The variables generated during bit-blasting are `BOOLEAN_TERM_VARIABLE`s and are not counted as Boolean variables by `CnfStream`.